src/main/native/cpp/view/AnalyzerPlot.cpp

```diff
@@ -278,4 +327,27 @@ void AnalyzerPlot::DisplayTimeDomainPlots() {
       }
     }
   }
+  if (m_fitNextPlot[6]) {
+    ImPlot::SetNextPlotLimits(0, m_dtMaxTime, 0, 50);
```

Comment:
Can we add a fit call just for the x-axis after this? See the arguments here: https://github.com/epezent/implot/blob/master/implot.h#L522